src/main/java/teammates/common/datatransfer/StudentAttributes.java

```diff
@@ -388,16 +388,22 @@ public Date getUpdatedAt() {
     /**
      * Should only be used for testing
      **/
+    //CHECKSTYLE:OFF 
+    // Naming violation is so that it will be noticed if it is accidentally included in production code 
     public void setCreated_NonProduction(Date createdAt) {
         this.createdAt = createdAt;
     }
+    //CHECKSTYLE:ON
 
     /**
      * Should only be used for testing
      **/
+    //CHECKSTYLE:OFF 
+    // Naming violation is so that it will be noticed if it is accidentally included in production code
     public void setUpdatedAt_NonProduction(Date updatedAt) {
         this.updatedAt = updatedAt;
     }
+    //CHECKSTYLE:ON
```

Comment:
Unrelated to this PR, but is it a better approach to create a subclass of `StudentAttributes` (or `*Attributes` for that matter) for testing only, and load them with these non-production methods? The subclass can reside as an inner class in `*AttributesTest` so they won't be used anywhere else.